README.md

```diff
@@ -418,6 +418,9 @@ Check that certain conditions are met in the provided arguments.
 If `fn` throws or returns a non-truthy value, show the thrown error, usage information, and
 exit.
 
+`global` indicates whether or not `check()` should be enabled both
```

Comment:
nit: I think 

> indicates whether `check()` should be enabled...

reads a little clearer, since I don't have to think about negations and my pyrate brain don't hurt.